packages/dashboard/src/pages/payments.js

```diff
@@ -19,22 +16,23 @@ const ActiveBadge = () => {
   );
 };
 
-export const getServerSideProps = authServerSideProps(async (context, api) => {
-  const [user, stats, stripe] = await Promise.all([
-    api.get("user").json(),
-    api.get("user/stats").json(),
-    api.get("stripe/prices").json(),
-  ]);
-  const plans = [config.tiers.starter, ...stripe].sort((a, b) => a.tier - b.tier);
+export default function Payments() {
+  const { data: user } = useAccountsApi("user");
+  const { data: stats } = useAccountsApi("user/stats");
+  const { data: prices } = useAccountsApi("stripe/prices");
```

Comment:
Q: My React is very rusty at best and non-existent at worst, so I have a question about promises here:
Are these executed sequentially or in parallel? `GET /user/stats` is not exactly fast - there's a lot of work to be done for it. If we want to improve the UX we should be doing these in parallel.